hazelcast-all/pom.xml

```diff
@@ -149,29 +149,26 @@
             <artifactId>hazelcast</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <!-- Make sure that the core module is the first dependency! -->
-        <!--        https://github.com/hazelcast/hazelcast/issues/15576-->
-        <!--dependency>
+        <dependency>
             <groupId>com.hazelcast</groupId>
             <artifactId>hazelcast-aws</artifactId>
-            <version>2.4</version>
-        </dependency-->
+            <version>3.0</version>
+        </dependency
```

Comment:
Missing bracket here, thus not compiling.

Reply:
fixed 👍